### PR TITLE
VS-2778 - gallery view design

### PR DESCRIPTION
**Pull request by bekand**

### CODE REVIEW

File: react/features/filmstrip/components/native/styles.js [hunk 1]
```diff
@@ -152,7 +150,7 @@ ColorSchemeRegistry.register('Thumbnail', {
      * Coloring if the thumbnail background.
      */
     participantViewStyle: {
-        backgroundColor: schemeColor('background')
```
Comment (molnartimi) [inline react/features/filmstrip/components/native/styles.js:155]: Unnecessary change

File: react/features/base/color-scheme/defaultScheme.js [hunk 2]
```diff
@@ -47,8 +47,8 @@ export default {
         background: 'rgb(249, 249, 249)'
     },
     'Thumbnail': {
-        activeParticipantHighlight: 'rgb(81, 214, 170)',
-        activeParticipantTint: 'rgba(49, 183, 106, 0.3)',
+        activeParticipantHighlight: 'rgb(243, 146, 160)',
+        activeParticipantTint: 'rgb(243, 146, 160)', // cabi pink?
```
Comment (bruderbarna) [inline react/features/base/color-scheme/defaultScheme.js]: Why the question mark in the comment? Also, could this be extracted to a constant somewhere? This is a pretty common color, used in foreseeably in many places.

File: react/features/filmstrip/components/native/TileView.js [hunk 3]
```diff
@@ -145,7 +146,7 @@ class TileView extends Component<Props> {
             return 2;
         }
 
-        return Math.min(3, participantCount);
+        return Math.min(3, participantCount);*/
```
Comment (bruderbarna) [inline react/features/filmstrip/components/native/TileView.js]: Why not just remove the code instead of commenting it out? If we need to get it back, it's in git history.

File: react/features/filmstrip/components/native/Thumbnail.js [hunk 4]
```diff
@@ -127,22 +127,25 @@ function Thumbnail(props: Props) {
             onClick = { _onClick }
             style = { [
                 styles.thumbnail,
-                participant.pinned && !tileView
-                    ? _styles.thumbnailPinned : null,
-                props.styleOverrides || null
+                props.styleOverrides || null,
+                renderDominantSpeakerIndicator ? _styles.dominantSpeaker : null
```
Comment (vetrabmercedes) [inline react/features/filmstrip/components/native/Thumbnail.js]: Please start boolean names with "is" prefix. For example `isDominantSpeaker`.

File: react/features/filmstrip/components/native/TileView.js [hunk 5]
```diff
@@ -131,21 +131,7 @@ class TileView extends Component<Props> {
...
-            return participantCount >= 3 ? 2 : 1;
-        }
-
-        if (participantCount === 4) {
-            // In wide view, a four person call should display as a 2x2 grid.
-            return 2;
-        }
-
-        return Math.min(3, participantCount);
+        return 2;
```
Comment (vetrabmercedes) [inline react/features/filmstrip/components/native/TileView.js]: Because we remove the whole function, maybe a constant for the `COLUMN_COUNT` will be better, than a `_getColumnCount()` function.

File: react/features/filmstrip/components/native/Thumbnail.js [hunk 6]
```diff
@@ -127,22 +127,25 @@ function Thumbnail(props: Props) {
...
             ] }
             touchFeedback = { false }>
 
             <ParticipantView
                 avatarSize = { tileView ? AVATAR_SIZE * 1.5 : AVATAR_SIZE }
                 participantId = { participantId }
                 style = { _styles.participantViewStyle }
-                tintEnabled = { participantInLargeVideo && !disableTint }
+                tintEnabled = { !disableTint }
```
Comment (vetrabmercedes) [inline react/features/filmstrip/components/native/Thumbnail.js]: Please start boolean names with "is" prefix. For example `isTintDisabled`.

**Review by molnartimi — APPROVED** [review]

File: react/features/base/avatar/components/native/StatelessAvatar.js [hunk 7]
```diff
@@ -161,7 +161,8 @@ export default class StatelessAvatar extends AbstractStatelessAvatar<Props> {
                 onError = { onAvatarLoadError }
                 resizeMode = 'cover'
                 source = {{ uri: url }}
-                style = { styles.avatarContent(size) } />
+                style = { styles.avatarContent(size) }
```
Comment (zongolista) [inline react/features/base/avatar/components/native/StatelessAvatar.js]: unnecessary change

File: react/features/base/avatar/components/native/styles.js [hunk 8]
```diff
@@ -25,7 +24,7 @@ export default {
     avatarContent: (size: number = DEFAULT_SIZE) => {
         return {
             height: size,
-            width: size
+            width: size,
```
Comment (zongolista) [inline react/features/base/avatar/components/native/styles.js]: unnecessary change

**Review by kecskesk — APPROVED** [review]

File: react/features/filmstrip/components/native/Thumbnail.js [hunk 9]
```diff
@@ -1,29 +1,29 @@
 // @flow
-
 import React from 'react';
 import { View } from 'react-native';
+import LinearGradient from 'react-native-linear-gradient';
 import type { Dispatch } from 'redux';
 
+
```
Comment (zongolista) [inline react/features/filmstrip/components/native/Thumbnail.js]: let's not leave empty lines here and there

File: react/features/filmstrip/components/native/Thumbnail.js [hunk 10]
```diff
@@ -1,29 +1,29 @@
...
 import { connect } from '../../../base/redux';
 import { StyleType } from '../../../base/styles';
 import { getTrackByMediaTypeAndParticipant } from '../../../base/tracks';
 import { DisplayNameLabel } from '../../../display-name';
 import { toggleToolboxVisible } from '../../../toolbox/actions.native';
-
-import DominantSpeakerIndicator from './DominantSpeakerIndicator';
 import styles, { AVATAR_SIZE } from './styles';
 
+
```
Comment (zongolista) [inline react/features/filmstrip/components/native/Thumbnail.js:24]: let's not leave empty lines here and there

File: react/features/filmstrip/components/native/styles.js [hunk 11]
```diff
@@ -123,16 +127,18 @@ export default {
...
     tileViewRows: {
-        justifyContent: 'center'
+        justifyContent: 'flex-start'
     },
 
     tileViewRow: {
         flexDirection: 'row',
-        justifyContent: 'center'
+        justifyContent: 'flex-start',
+
```
Comment (zongolista) [inline react/features/filmstrip/components/native/styles.js]: let's not leave empty lines here and there

**Review by zongolista — APPROVED** [review]

**Review by vetrabmercedes — APPROVED** [review]
